Update enroll-a-windows-10-device-automatically-using-group-policy.md (#4969)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy.md

    ![MDM policies](images/autoenrollment-mdm-policies.png)

4. Double-click **Enable Automatic MDM enrollment using default Azure AD credentials**.

    ![MDM autoenrollment policy](images/autoenrollment-policy.png)

5. Click **Enable**, then click **OK**.

>[!IMPORTANT]

> >In Windows 10, version 1903, the MDM.admx file was updated to include the possibility to select which credential is used to enroll the device. The **Device Credential** is a new option that will only have effect on clients that have the Windows 10 1903 feature update installed. 

Reviewer: ```suggestion
>In Windows 10, version 1903, the MDM.admx file was updated to include an option to select which credential is used to enroll the device. **Device Credential** is a new option that will only have an effect on clients that have the Windows 10, version 1903 feature update installed. 
```

> >The default behaviour for older releases is to fallback to **User Credential**.

Reviewer: ```suggestion
>The default behavior for older releases is to revert to **User Credential**.
```

Author: Thanks Johan :)

Reviewer 2: Heh. Should have known that MS docs preferably use EN-US spelling instead of EN-GB.

    ![MDM policies](images/autoenrollment-mdm-policies.png)

4. Double-click **Enable Automatic MDM enrollment using default Azure AD credentials**.

    ![MDM autoenrollment policy](images/autoenrollment-policy.png)

5. Click **Enable**, then click **OK**.

> >[!NOTE]

Reviewer 2: Recommended MarkDown codestyle: 1 space between quote indicator and its following text.
```suggestion
> [!NOTE]
```
(this is not an absolute, but recommended for compatibility)

Author: You would have to go through a lot of MS docs to make this consistent. :) 
 I am just copying what was already there.

> >In Windows 10, version 1903, the MDM.admx file was updated to include an option to select which credential is used to enroll the device. **Device Credential** is a new option that will only have an effect on clients that have the Windows 10, version 1903 feature update installed. 

Reviewer 2: (same here)
```suggestion
> In Windows 10, version 1903, the MDM.admx file was updated to include an option to select which credential is used to enroll the device. **Device Credential** is a new option that will only have an effect on clients that have the Windows 10, version 1903 feature update installed. 
```

> The default behaviour for older releases is to fallback to **User Credential**.

Reviewer: ```suggestion
The default behavior for older releases is to revert to **User Credential**.
```

> When a group policy refresh occurs on the client, a task is created and scheduled to run every 5 minutes for the duration of 1 day. The task is called " Schedule created by enrollment client for automatically enrolling in MDM from AAD." 

Reviewer: ```suggestion
When a group policy refresh occurs on the client, a task is created and scheduled to run every 5 minutes for the duration of one day. The task is called " Schedule created by enrollment client for automatically enrolling in MDM from AAD." 
```

To see the scheduled task, launch the [Task Scheduler app](#task-scheduler-app).

If two-factor authentication is required, you will be prompted to complete the process. Here is an example screenshot.

![Two-factor authentication notification](images/autoenrollment-2-factor-auth.png)

> >[!Tip]

Reviewer 2: MD codestyle:
```suggestion
> [!Tip]
```

> > You can avoid this behaviour using Conditional Access Policies in Azure AD.

Reviewer: ```suggestion
> You can avoid this behavior by using Conditional Access Policies in Azure AD.
```

- Ensure that PCs belong to same computer group.

>[!IMPORTANT]

> >If you do not see the policy, it may be because you don’t have the ADMX installed for Windows 10, version 1803, version 1809 or version 1903. To fix the issue, follow these steps (Note: the latest MDM.admx is backwards compatible):        

Reviewer: ```suggestion
>If you do not see the policy, it may be because you don’t have the ADMX installed for Windows 10, version 1803, version 1809, or version 1903. To fix the issue, follow these steps (Note: the latest MDM.admx is backwards compatible):        
```

>   1. Download:  
>   1803 -->[Administrative Templates (.admx) for Windows 10 April 2018 Update (1803)](https://www.microsoft.com/download/details.aspx?id=56880) or  
>   1809 --> [Administrative Templates for Windows 10 October 2018 Update (1809)](https://www.microsoft.com/download/details.aspx?id=57576).

> >   1903 --> [Administrative Templates for Windows 10 May 2019 Update (1903)](https://www.microsoft.com/en-us/download/details.aspx?id=58495&WT.mc_id=rss_alldownloads_all) (recommended).

Reviewer 2: Avoid country-specific parts in the links, unless the page is guaranteed never to be translated to other languages.
```suggestion
>   1903 --> [Administrative Templates for Windows 10 May 2019 Update (1903)](https://www.microsoft.com/download/details.aspx?id=58495&WT.mc_id=rss_alldownloads_all) (recommended).
```

>   2. Install the package on the Primary Domain Controller (PDC).
>   3. Navigate, depending on the version to the folder:
>   **C:\Program Files (x86)\Microsoft Group Policy\Windows 10 \<month\> \<year\> Update (\<feature release\>) \<policy version\>**

> >   4. Copy policy definitions folder to **C:\Windows\SYSVOL\domain\Policies** (N.B. If this folder does not exist, then be aware that you will be switching to a [central policy store](https://support.microsoft.com/da-dk/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administra) for your entire domain).

Reviewer 2: Same here:
```suggestion
>   4. Copy policy definitions folder to **C:\Windows\SYSVOL\domain\Policies** (N.B. If this folder does not exist, then be aware that you will be switching to a [central policy store](https://support.microsoft.com/help/3087759/how-to-create-and-manage-the-central-store-for-group-policy-administra) for your entire domain).
```

- Ensure that PCs belong to same computer group.

>[!IMPORTANT]

> >If you do not see the policy, it may be because you don’t have the ADMX installed for Windows 10, version 1803, version 1809, or version 1903. To fix the issue, follow these steps (Note: the latest MDM.admx is backwards compatible):        

Reviewer 2: MD codestyle:
```suggestion
> If you do not see the policy, it may be because you don’t have the ADMX installed for Windows 10, version 1803, version 1809, or version 1903. To fix the issue, follow these steps (Note: the latest MDM.admx is backwards compatible):        
```
